src/Naderman/Composer/AWS/AwsClient.php

```diff
@@ -35,6 +36,16 @@ class AwsClient
     protected $io;
 
     /**
+     * @var S3Client
+     */
+    protected $client;
```

Comment:
Seems like this is for tests only, can't you refactor the tests so this isn't needed?